exercises/selection_sort/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+//selection sort
+func Sort(s []int) {
```

Comment:
this algorithm should not swap multiple times. it should find the smallest (or the largest, depending on your implementation) and produce only one swap. pseudo-code would look like this:
```
for smallest in indices of s {
  index of min = find index of min in s[smallest:]
  if index of min is not smallest {
    swap index of min and smallest in s
  }
}

```
it's important there is only one swap